cdap-data-fabric/src/main/java/co/cask/cdap/spi/data/StructuredTableAlreadyExistsException.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.spi.data;
+
+import co.cask.cdap.common.AlreadyExistsException;
+import co.cask.cdap.spi.data.table.StructuredTableId;
+
+/**
+ * Exception thrown when a structured table already exists.
+ */
+public class StructuredTableAlreadyExistsException extends AlreadyExistsException {
```

Comment:
Do we need this subclass? Or can we just reuse AlreadyExistsException?

Reply:
Removed this class